src/m_menu.c

```diff
@@ -4185,6 +4185,7 @@ static const char *layout_strings[] = {
 };
 
 static const char *curve_strings[] = {
+  "", "", "", "", "", "", "", "", "", "", // Dummy values, start at 1.0.
```

Comment:
I wonder if there's a better way to do this.

Reply:
This solution is alright.

This solution is fine.

It's not that important what indexes are stored in config. For most users, `(10 = Linear, 20 = Squared, 30 = Cubed)` is as cryptic as `(0 = Linear, 10 = Squared, 20 = Cubed)`. Printing and reading floats is challenging (see https://netlib.org/fp/dtoa.c) and, as @fabiangreffrath points out, there are still cross-platform issues. I don't think we need sophisticated UI widgets with float support for the Doom menu.

Reply:
Sounds good to me. Should we move forward with these cosmetic changes? "10/20/30" is slightly more intuitive for me, and should make looking at this code a little less cryptic in the future. It's not critical though.

Reply:
> Should we move forward with these cosmetic changes?

Sure. If it doesn't affect user configs from the previous version, I'm fine with any cosmetic change.